docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_grafana/_index.md

```diff
@@ -0,0 +1,170 @@
+- **[Vagrant Windows box](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_servers/vagrant/local_vagrant_windows/)**
+
+Please review the [Azure Monitor Agent (AMA) supported OS documentation](https://docs.microsoft.com/azure/azure-monitor/agents/agents-overview#supported-operating-systems) and ensure that the VMs you will use for this exercise are supported.
+
+## Prerequisites
+
+- As mentioned, this scenario starts at the point where you already deployed and connected VMs or servers to Azure Arc. In the screenshots below, you can see a Windows server that have been connected with Azure Arc and are visible as a resource in Azure.
+
+    ![Screenshot Azure Arc-enabled servers on resource group](./01.png)
+
+    ![Screenshot Windows Azure Arc-enabled server connected status](./02.png)
+
+- [Install or update Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli?view=azure-cli-latest). Azure CLI should be running version 2.14 or later. Use ```az --version``` to check your current installed version.
+
+## Deployment Options and Automation Flow
+
+This Jumpstart scenario provides multiple paths for deploying and configuring resources. Deployment options include:
+
+- Azure portal
+- ARM template via Azure CLI
+
+The steps below will help you get familiar with the automation and deployment flow.
+
+1. User provides the ARM template parameter values, either via the portal or editing the parameters file. These parameter values are being used throughout the deployment.
+
+2. User deploys the ARM template at the resource group level.
+
+3. User is verifying the successful deployment of Log Analytics workspace, Azure Managed Grafana, AMA extension, and data collection rules creation.
+
+## Deployment Option 1: Azure portal
+
+- Click the <a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fmicrosoft%2Fazure_arc%2Fmain%2Fazure_arc_servers_jumpstart%2Fgrafana%2Fgrafana-template.json" target="_blank"><img src="https://aka.ms/deploytoazurebutton"/></a> button and enter values for the the ARM template parameters.
+
+  ![Screenshot showing Azure portal deployment](./03.png)
+
+Use the following command to get the AAD id of the current signed in user and copy it to the Grafana Adminprincipal Id parameter:
+
+```shell
+az ad signed-in-user show --query id -o tsv
+```
+
+  ![Screenshot showing Azure portal deployment](./04.png)
+
+## Deployment Option 2: ARM template with Azure CLI
+
+As mentioned, this deployment will leverage ARM templates.
+
+- Clone the Azure Arc Jumpstart repository
+
+    ```shell
+    git clone https://github.com/microsoft/azure_arc.git
+    ```
+
+- Edit the [*parameters file*](https://github.com/microsoft/azure_arc/blob/main/azure_arc_servers_jumpstart/grafana/grafana-template.parameters.json) providing the values that match your configuration as described above.
```

Comment:
For italic text, use "_" instead of "*"-. Review all the occurrences
```suggestion
- Edit the [_parameters file_](https://github.com/microsoft/azure_arc/blob/main/azure_arc_servers_jumpstart/grafana/grafana-template.parameters.json) providing the values that match your configuration as described above.
```